apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector-item.js

```diff
@@ -54,8 +53,7 @@ const TemplateSelectorItem = props => {
 			type="button"
 			className="template-selector-item__label"
 			value={ value }
-			onMouseEnter={ () => onFocus( value, label ) }
-			onClick={ () => onSelect( value, label ) }
+			onClick={ () => onSelect( value ) }
```

Comment:
from PR description:

> removed long unused code for onFocus (which had nothing to do with a focus)

Reply:
I was standing between the choice to either update the function call to keep it working (the same way as `onClick`) or remove it because we don't use it at all. I chose the 🔥method but happy to revert if you want the fn to stay for some reason.